WORKSPACE

```diff
@@ -15,11 +15,11 @@
 #
 # A Bazel workspace for the IAP JWT verfication nginx module.
```

Comment:
I think this line needs to be updated as well. Maybe line 35 as well.

I'm not familiar with open source at Google. What is the purpose of this file?